src/util/triplea/mapXmlCreator/MapXmlCreator.java

```diff
@@ -303,7 +303,6 @@ public MapXmlCreator(final boolean testMode) {
       try {
         loadXmlFromFilePath(myFile.getAbsolutePath());
       } catch (SAXException | IOException | ParserConfigurationException e) {
-        // TODO Auto-generated catch block
         log(Level.SEVERE, "Default Map XML File could not be loaded from '" + myFile.getAbsolutePath() + "'.");
```

Comment:
Thats not the Problem...
According to the JavaDoc of Level
`SEVERE (highest value)
WARNING
INFO
CONFIG
FINE
FINER
FINEST  (lowest value)`
Are the only options... this is a Java internal thing...